server/modules/surveyRdb/repository/nodeAnalysisTableRepository.js

```diff
@@ -0,0 +1,33 @@
+import * as SchemaRdb from '@common/surveyRdb/schemaRdb'
+import * as NodeAnalysisTable from '@common/surveyRdb/nodeAnalysisTable'
+import * as SurveySchemaRepositoryUtils from '@server/modules/survey/repository/surveySchemaRepositoryUtils'
+
+export const createNodeAnalysisTable = async (surveyId, client) => {
+  const schemaRdb = SchemaRdb.getName(surveyId)
+  const schemaSurvey = SurveySchemaRepositoryUtils.getSurveyDBSchema(surveyId)
+
+  await client.query(`
+    CREATE TABLE
+      ${schemaRdb}.${NodeAnalysisTable.tableName}
+    (
+      uuid                  uuid      NOT NULL,
+      processing_step_uuid  uuid      NOT NULL,
+      record_uuid           uuid      NOT NULL,
+      parent_uuid           uuid          NULL,
+      node_def_uuid         uuid      NOT NULL,
+      value                 jsonb     NOT NULL,
```

Comment:
I would add a default to uuid and value. or not?

Reply:
right, I'll do it